src/routes/+layout.svelte

```diff
@@ -69,6 +72,40 @@
 
   let navMenuExpanded = false;
   $: if ($navigating) navMenuExpanded = false;
+
+  // This is a workaround for https://github.com/sveltejs/kit/issues/10122
+  // eslint-disable-next-line no-undef
+  let lastPageURL: URL | undefined, lastPageError: App.Error | null | undefined;
+  $: ({ url: lastPageURL, error: lastPageError } = $page || {});
+  beforeNavigate(async ({ to, type, cancel }) => {
+    const previousPreviewValue = lastPageURL?.searchParams.get("preview");
+    if (
+      typeof previousPreviewValue === "string" &&
+      type === "link" &&
+      to &&
+      !to.url.searchParams.has("preview") &&
+      !lastPageError &&
+      !previewAuthenticationError &&
+      !(to.url.pathname === "/login" || to.url.pathname === "/logout")
+    ) {
+      to.url.searchParams.set("preview", previousPreviewValue);
```

Comment:
small nit:  because of how `searchParams.set()` functions, all subsequent page loads include an "=" at the end, e.g. `ldaf.la.gov/some-sub-route?preview=`. It doesn't look like it would take a lot to update logic to use `preview=true` and then check for that, i.e., also prevent the confusing state of showing preview content even if the url has `?preview=false` at the end